### ISSUE TITLE
Lasers do 15 damage

### ISSUE THREAD

**Issue by Sarcalogo** [HippieStation/HippieStation13#226]
They used to do 20, now they do 15.

**Comment by PopNotes** [comment]
Do they? I thought lasers felt a little weak.

**Comment by banthebantz** [comment]
That's fucking gay

**Comment by KittyKathy** [comment]
This is wrong they do 20 damage to unarmored targets, but people like the captain and hos start off with gear that has armor values. So if you try to laser the captain or hos the damage is reduced. There also seems to be a bug with armor that only covers chest/groin area where it would do full damage if it hits your chest as if it hits your leg or a unarmored part, this does not seem to be the case for the hos armored coat which protects the whole body but the head equally.
Only thing I see that needs a damage change is pulses which do 50 damage to unarmored targets it used to be 40 in pre rebase I think.
